### ISSUE TITLE
Opacity animations don't work on the web with html renderer

### ISSUE THREAD

**Issue by behnamsattar** [rive-app/rive-flutter#200]
### Description
Opacity animations don't work on the web!

### Steps To Reproduce
1. Create an animation on the opacity values
2. Run it on the flutter web and see how it doesn't work.

#### Source `.riv`/`.rev` file
[Archive.zip](https://github.com/rive-app/rive-flutter/files/7929517/Archive.zip)

### Expected behavior
Try the animation in the editor and see how it works with different triggers. Notice that there are a bunch of elements that are hidden (zero opacity) in the neutral state. I expect the animation to work in the same way on the web implementation in Flutter. However, it doesn't hide the zero opacity elements and it seems like the opacity changes during animations don't work.

### Device & Versions
- Device: Chrome (Mac)
- OS: Web 
- Flutter Version: 
```
Flutter 2.8.1 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 77d935af4d (6 weeks ago) • 2021-12-16 08:37:33 -0800
Engine • revision 890a5fca2e
Tools • Dart 2.15.1
```

**Comment by zplata** [comment]
Hey @behnamsattar, a few follow-ups:
- What version of `rive-flutter` are you working with?
- If it's been a while since you last exported the `.riv` file, have you tried re-exporting the file?
- Also, do you have a file I can reference to see how you're integrating the Rive file, or a snippet of code you can paste?

I pulled down your `.rev` file and re-exported it from a new file in my editor, ran that `.riv` on a small flutter web app, and seem to be getting the same result as what I see in the Rive editor with the changing opacity. For reference, here's my snippet of code, run on `rive: ^0.8.1`.

```
  SMITrigger? _relating;
  SMITrigger? _neutral;
  SMITrigger? _inspiration;
  SMITrigger? _laughing; 

  void _onRiveInit(Artboard artboard) {
    StateMachineController? controller =
        StateMachineController.fromArtboard(artboard, 'State Machine');
    artboard.addController(controller!);
    _relating = controller.findInput<bool>('Relating') as SMITrigger;
    _neutral = controller.findInput<bool>('Neutral') as SMITrigger;
    _inspiration = controller.findInput<bool>('Inspiration') as SMITrigger;
    _laughing = controller.findInput<bool>('Laughing') as SMITrigger;
  }
  
  @override
  Widget build(BuildContext context) {
    return Scaffold(
        backgroundColor: Colors.black,
        appBar: AppBar(
          title: const Text('Test'),
        ),
        body: Center(
          child: Column(
              mainAxisAlignment: MainAxisAlignment.center,
              crossAxisAlignment: CrossAxisAlignment.center,
              children: <Widget>[
                SizedBox(
                  width: 200,
                  height: 174,
                  child: RiveAnimation.asset(
                    'assets/flutter1.riv',
                    onInit: _onRiveInit,
                  ),
                ),
                TextButton(
                  onPressed: () {
                    _relating!.fire();
                  },
                  child: const Text('Relating'),
                ),
                TextButton(
                  onPressed: () {
                    _neutral!.fire();
                  },
                  child: const Text('Neutral'),
                ),
                TextButton(
                  onPressed: () {
                    _inspiration!.fire();
                  },
                  child: const Text('Inspiration'),
                ),
                TextButton(
                  onPressed: () {
                    _laughing!.fire();
                  },
                  child: const Text('Laughing'),
                ),
              ]),
        ));
  }

**Comment by behnamsattar** [comment]
Hey @zplata,

- I'm working with `rive: ^0.8.1`.
- I exported the file today on the editor version `Beta 0.7.1`.
- I just used the same piece of code you have on a new project and saw that in fact it is working. After digging more thought I figured the issue is there only when the app is rendered using the html renderer. So if you try running `flutter run --web-renderer html` you'll see that non of the opacity animations work. I have screenshots attached comparing both.

<img width="591" alt="Screen Shot 2022-01-24 at 8 09 36 PM" src="https://user-images.githubusercontent.com/29583601/150909590-36fd98b0-c259-4801-ad5e-1eb9fbbec01b.png">
<img width="427" alt="Screen Shot 2022-01-24 at 8 11 40 PM" src="https://user-images.githubusercontent.com/29583601/150909640-b224dfe7-d4d4-4c93-a386-f957d5cdc234.png">

You can see how the hidden elements show up in the html renderer version.

**Comment by zplata** [comment]
Gotcha, thanks for clarifying that! Chatted with the team a bit around this and unfortunately, this is a bug at the Flutter engine level for that renderer, beyond the Rive layer here. For now, I’d suggest running with the CanvasKit renderer to support your needs in this case.

**Comment by behnamsattar** [comment]
Thanks for getting to the bottom of this, @zplata. I'll file an issue on Flutter's repository and reference this issue. Meanwhile, I'll try to develop a workaround on the HTML renderer. Unfortunately, using CanvasKit is not an option for the company due to other issues.

**Comment by caudetgit** [comment]
https://github.com/flutter/flutter/issues/97237#issuecomment-1035368982

**Comment by HayesGordon** [comment]
This fix has landed in v0.10.2
